src/test/shell/integration/java_integration_test.sh

```diff
@@ -166,12 +166,16 @@ EOF
 
 # This test intentionally show some errors on the standard output.
 function test_compiles_hello_world() {
+  declare -a bazel_opts=()
+  if [[ $# -gt 0 ]]; then
+    bazel_opts=("${@}")
```

Comment:
why not just always add this?

Reply:
I was trying to avoid an unbound variable error but I still wasn't successful for old bash versions (used in Ubuntu 14,16 and on MacOS)... Fixed it now, although it looks uglier :slightly_frowning_face: